css/pages/home.css

```diff
@@ -32,6 +32,57 @@
 /* end of Testimonials */
 
 /* Plans */
+.center {
+  text-align: center;
+}
+.container > .row {
+  margin-right: 0;
+  padding-right: 0;
+}
+.heading {
+  font-size: 40px;
+  text-align: center;
+  color: #222;
+  margin-top: 20px;
+}
+.heading2 {
+  font-size: 22px;
+  text-align: center;
+  color: #222;
+  margin-top: 20px;
+}
+.text {
+  font-size: 17px;
+  text-align: center;
+  color: #777;
+}
+.center.plan {
+  background-color: #f9f9ff;
+  padding: 40px 20px;
+  margin: 20px 0;
+}
+.plan p:first-of-type {
+  padding-bottom: 30px;
+}
+.plan > p {
+  font-size: 17px;
+  text-align: center;
+  color: #777;
+  padding: 12px;
+  border-bottom: 1px solid #ddd;
+}
+.circle {
```

Comment:
Dar pridečiau line-height: 52px;, kad numeriukas lygiai per viduri butu.

Reply:
na as centravimo sprendima pasirinkau spresti su paddingu,bet galima aisku ir su line-heightu..tada paddingo net neverta deti, nes jie abu turi ta pati efekta. Su padingu auksciui lieka tik 50px idealiam centravimui, be padingo line-height 80px.